sei-devnet-2/genesis.json

```diff
@@ -0,0 +1,1475 @@
+{
+  "genesis_time": "2023-01-11T22:00:00.000000000Z",
+  "chain_id": "sei-devnet-2",
+  "initial_height": "1",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "22020096",
+      "max_gas": "-1"
+    },
+    "evidence": {
+      "max_age_num_blocks": "100000",
+      "max_age_duration": "172800000000000",
+      "max_bytes": "1048576"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    },
+    "version": {
+      "app_version": "0"
+    },
+    "synchrony": {
+      "precision": "505000000",
+      "message_delay": "12000000000"
+    },
+    "timeout": {
+      "propose": "1000000000",
+      "propose_delta": "500000000",
+      "vote": "50000000",
+      "vote_delta": "500000000",
+      "commit": "50000000",
+      "bypass_commit_timeout": false
+    },
```

Comment:
interesting - do you know if these are ignored in favor of config.toml?

Reply:
i believe the fields in config.toml are meant to override these as it's set for all validators whereas config.toml can be configured to be overridden per validator?